python/lsst/ts/attcpip/csc.py

```diff
@@ -129,12 +132,74 @@ def get_config_pkg() -> str:
     def connected(self) -> bool:
         return self.cmd_evt_client.connected and self.telemetry_client.connected
 
-    async def handle_summary_state(self) -> None:
-        """Called when the summary state has changed."""
-        if self.summary_state in (salobj.State.DISABLED, salobj.State.ENABLED):
-            await self.start_clients()
+    async def begin_disable(self, data: salobj.BaseMsgType) -> None:
+        """Begin do_disable; called before state changes.
+
+        Parameters
+        ----------
+        data : `DataType`
+            Command data
+        """
+        command = CommonCommand.DISABLE
+        if self.connected:
+            command_issued = await self.write_command(command=command)
+            await command_issued.done
+        else:
+            self.log.warning(f"Not connected so not sending the {command} command.")
+
+    async def begin_enable(self, data: salobj.BaseMsgType) -> None:
+        """Begin do_enable; called before state changes.
+
+        Parameters
+        ----------
+        data : `DataType`
+            Command data
+        """
+        command = CommonCommand.ENABLE
+        if self.connected:
+            command_issued = await self.write_command(command=command)
+            await command_issued.done
+        else:
+            self.log.warning(f"Not connected so not sending the {command} command.")
+
+    async def begin_standby(self, data: salobj.BaseMsgType) -> None:
+        """Begin do_standby; called before the state changes.
+
+        Parameters
+        ----------
+        data : `DataType`
+            Command data
+        """
+        command = CommonCommand.STANDBY
+        if self.connected and self.summary_state in [
+            salobj.State.DISABLED,
+            salobj.State.ENABLED,
+        ]:
```

Comment:
Is the `standby` command related to the `Enabled` state? I thought it is only related to `Disabled` state?